examples/weather/README.md

```diff
@@ -1,40 +1,40 @@
-[Back to DS](../../README.md)
+[Back to Datashare](../../README.md)
 
-# Example scenario using weather observation data
+# Example scenario using sample weather observation data
 
 ## Overview
-For this DS example, we configure and load weather observation
-data. While the raw data used in this scenario contains many columns,
-we are going to restrict the transformed data to only three columns: a
+For this Datashare example, you configure and load sample weather observation 
+data. While the source data in this example contains many columns,
+the data is being reduced to share only three columns: the observation
 timestamp, the measurement's coordinates as a `GEOGRAPHY` type (manufactured by calling the
 [ST_GeogPoint](https://cloud.google.com/bigquery/docs/reference/standard-sql/geography_functions#st_geogpoint)
-function), and the mean temperature that comprises the observation. The
-destination table will also contain the `cds_batch_id` column, that
-is added automatically by `DS`, so that you may
-associate the table data with its specific ingestion cycle.
+function) column, and the mean temperature that comprises the observation. Datashare will also add a `cds_batch_id` column automatically, to associate groups of rows in the table with their specific ingestion cycle.
```

Comment:
Should we change this ingestion function so that the processPrefix is now 'ds', and that the batch id will than be ds_batch_id?

Reply:
I'd argue for `datashare_batch` or `datashare_batch_id` or even `datashare_id`.

Reply:
or even simply `batch_id`, but a prefix is probably advisable.

Reply:
I prefer datashare_batch_id

Reply:
`datashare_batch_id` it is - will open up ticket to patch ingestion function